Optimizing Database audit logging (#20565)
pingcap/docs: tidb-cloud/tidb-cloud-auditing.md

TiDB Cloud supports recording the audit logs of a TiDB Cloud Dedicated cluster to your cloud storage service. Before enabling database audit logging, configure your cloud storage service on the cloud provider where the cluster is located.

> **Note:**
>

> > - For AWS deployment scenarios, When you enable Database audit logging, you can choose to store audit log files in TiDB Cloud.
> > - Currently, this feature is only available upon request. To request this feature, click ? in the lower-right corner of the TiDB Cloud console and click Request Support. Then, fill in "Apply to store audit log files in TiDB Cloud" in the Description field and click Submit.

Reviewer: ```suggestion
> For TiDB clusters deployed on AWS, you can choose to store audit log files in TiDB Cloud when enabling database audit logging. Currently, this feature is only available upon request. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Request Support**. Then, fill in "Apply to store audit log files in TiDB Cloud" in the **Description** field and click **Submit**.
```

## Specify auditing filter rules

> After enabling audit logging, you must specify auditing filter rules to control which user access events to capture and write to audit logs versus. If no filter rules are specified, TiDB Cloud does not log anything.

Reviewer: ```suggestion
After enabling audit logging, you must specify auditing filter rules to control which user access events to capture and write to audit logs. If no filter rules are specified, TiDB Cloud does not log anything.
```

To specify auditing filter rules for a cluster, take the following steps:

> 1. On the **DB Audit Logging** page, you can click **Add Filter Rule** in the **Log Filter Rules** section to add audit filter rules.
> 
>     You can add one audit rule at a time. Each rule specifies a user expression, database expression, table expression, and access type. You can add multiple audit rules to meet your audit requirements.
> 
> 2.In the **Log Filter Rules** section, click **>** to view the audit rules that have been added.

Reviewer: ```suggestion
1. On the **DB Audit Logging** page, click **Add Filter Rule** in the **Log Filter Rules** section to add an audit filter rule.

    You can add one audit rule at a time. Each rule specifies a user expression, database expression, table expression, and access type. You can add multiple audit rules to meet your auditing requirements.

2.In the **Log Filter Rules** section, click **>** to expand and view the list of audit rules you have added.
```

## View audit logs

> TiDB Cloud stores the audit log files of the database in your storage service by default, so you need to read the audit log information in your storage service.

Reviewer: ```suggestion
By default, TiDB Cloud stores database audit log files in your storage service, so you need to read the audit log information from your storage service.
```

> **Note:**
>

> > If you have applied and chosen to store audit log files in TiDB Cloud, you can download your audit log files in the **Audit Log Access** section of the **Database Audit Logging** page.

Reviewer: ```suggestion
> If you have requested and chosen to store audit log files in TiDB Cloud, you can download them from the **Audit Log Access** section on the **Database Audit Logging** page.
```
